CHANGELOG.md

```diff
@@ -1,21 +1,23 @@
 # Blip
-Blip is the web front end for YourLoops system.
+Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
 ## 1.4.2 - 2020-09-09  
 ### Fixed
 - YLP-157 Add german locale to the list of available languages.
+### Engineering
```

Comment:
This version seems to be already released.
Can you put the change in an "Unreleased" tag ?
And maybe also make less line change in the file ?